sdc-plugin/buffers.h

```diff
@@ -51,13 +51,8 @@ struct Pll {
 	clk_mult = FetchParam(cell, "CLKFBOUT_MULT", 5.0);
 	divclk_divisor = FetchParam(cell, "DIVCLK_DIVIDE", 1.0);
 	for (auto clk_output : outputs) {
-	    // CLKOUT[0-5]_DIVIDE
-	    clkout_divisors[clk_output] = FetchParam(cell, clk_output + "_DIVIDE", 1.0);
 	    clkout_period[clk_output] = CalculatePeriod(clk_output);
```

Comment:
`CalculatePeriod` depends on "CLKIN1_PERIOD" being valid.  Should we add an explicit check that the clock period of "CLKIN1_PERIOD" matches the virtual clock constraint propigated to that net?

Reply:
Added the check.